### ISSUE TITLE
add_task: unexpected interaction with `find_contact` when adding a task that isn't assigned to a contact

### ISSUE THREAD

**Issue by jontmy** [jontmy/ped#5]
```
clear
add_contact n/Alice A p/999 e/a@aa.aa a/aa
add_contact n/Bobby B p/999 e/b@bb.bb a/bb
find_contact A
add_task n/foo d/foo
```

Task is successfully added "New task added: [foo]", but nothing shows up:

![Screenshot 2022-10-28 at 16.42.41.png](https://raw.githubusercontent.com/jontmy/ped/master/files/b69d50fe-3560-400b-9d22-1988a2a91f92.png)

Only after a "list_task" will it show up:

![Screenshot 2022-10-28 at 16.43.59.png](https://raw.githubusercontent.com/jontmy/ped/master/files/eede1d07-4dfa-401d-8c47-992152ebf5c6.png)

This behavior (if intentional) is not documented anywhere.
